packages/expo-cli/src/commands/bundle-assets.ts

```diff
@@ -13,8 +13,8 @@ async function action(projectDir: string, options: Options) {
 export default function(program: Command) {
   program
     .command('bundle-assets [project-dir]')
-    .option('--dest [dest]', 'Destination directory for assets')
-    .option('--platform [platform]', 'detached project platform')
+    .option('--dest [dir]', 'Destination directory for assets')
+    .option('--platform [platform]', 'Detached project platform')
```

Comment:
We no longer use the word “detached” in documentation.
```suggestion
    .option('--platform [platform]', 'Platform [android or ios]')
```